src/modules/vendor/pages/ni/config/ActivityProfile.vue

```diff
@@ -75,6 +80,7 @@ export default {
     };
   },
   computed: {
+    ...mapState('program', ['program', 'activity']),
```

Comment:
du coup dans le created en dessous on pourrait mettre : " if(!this.program) await this.$store.dispatch('program/fetchProgram', { programId: this.programId });" au lieu de la ligne 97